plugins/l4d2/l4d2.cpp

```diff
@@ -3,10 +3,58 @@
 // that can be found in the LICENSE file at the root of the
 // Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
+#ifdef WIN32
 #include "../mumble_plugin_win32_32bit.h" // Include standard plugin header.
+#else
+#include "../mumble_plugin_linux_32bit.h" // Include standard plugin header.
+#endif
+
 #include "../mumble_plugin_utils.h" // Include plugin header for special functions, like "escape".
 
-procptr32_t serverid_steamclient, playerid_engine, player_server; // Variables to contain modules addresses
+// Variables to contain modules addresses
+procptr32_t steamclient = 0;
+procptr32_t server = 0;
+procptr32_t engine = 0;
+
+#ifdef WIN32
+// Memory addresses
+const procptr32_t state_address        = 0x6ACBD5;
+const procptr32_t avatar_pos_address   = 0x6B9E1C;
+const procptr32_t camera_pos_address   = 0x774B98;
+const procptr32_t avatar_front_address = 0x774BF8;
+const procptr32_t avatar_top_address   = 0x774C28;
+const procptr32_t host_address         = 0x772B24;
+const procptr32_t servername_address   = 0x772D2C;
+const procptr32_t map_address          = 0x772C28;
+const procptr32_t serverid_steamclient_offset = 0x95E56D;
+const procptr32_t player_server_offset        = 0x7F87BC;
+const procptr32_t playerid_engine_offset      = 0x4EBF88;
+// Module names
+const wchar_t * exe_name         = L"left4dead2.exe";
```

Comment:
The joys of switching between projects that use spaces and those that use tabs...

Fixing.